### DISCUSSION TITLE
Domoticz send MQTT command

### DISCUSSION THREAD

**Discussion by cpaolo** [HASwitchPlate/openHASP#600] [Q&A]
I love openHASP, but i'm a newbie, i cannot find the way for sending a MQTT command to Domoticz for switching a light (idx 1).
Should i put the command in the pages.json ?

**Comment by red-kooga** [comment]
The way I use openhasp with domoticz is via node red.

[flows.json](https://github.com/HASwitchPlate/openHASP/files/13808805/flows.json)

**Reply by cpaolo** [reply]
Thankyou very much, i need to learn because i don't know node red.

How can use this flow ? Just put it in node red ?

**Reply by red-kooga** [reply]
that is just an example mqtt message to send to domoticz/in, you need to subscribe to a the topic of your openhasp, example "hasp/lanbon/state/p1b2" in node red then create a function to send the message to domoticz.
steve cope has some great videos on youtube on how to use node red. 
https://youtu.be/XDrwgMSQrEY?si=m9G7oLSF62W-3NIL
